src/metabase/driver/crate.clj

```diff
@@ -0,0 +1,96 @@
+(ns metabase.driver.crate
+  (:require [metabase.driver :as driver]
+            [metabase.driver.generic-sql :as sql]
+            [clojure.set :as set]
+            (korma [core :as k])
+            (metabase.driver.crate [query-processor :as qp]
+                                   [util :as u]
+                                   [generic-sql :as gs]
+                                   [native :as n]))
+  (:import (clojure.lang Named)))
+
+(defn- column->base-type
+  "Map of Crate column types -> Field base types"
+  [_ column-type]
+  ({:integer          :IntegerField
+    :string           :TextField
+    :boolean          :BooleanField
+    :byte             :IntegerField
+    :short            :IntegerField
+    :long             :BigIntegerField
+    :float            :FloatField
+    :double           :FloatField
+    :ip               :TextField
+    :timestamp        :DateTimeField
+    :geo_shape        :DictionaryField
+    :geo_point        :ArrayField
```

Comment:
the `GEO_POINT` data type is an array of `[ <longitude>, <latitude> ]`
maybe there is a better way to display it instead of an array, e.g. as separate "virtual columns" for lat and lon which then may be used for the pin map visualisation